test/library/standard/Path/victor-ludorum/testcases_isAbsPath.chpl

```diff
@@ -0,0 +1,15 @@
+   use Path;
```

Comment:
This test will currently generate the error message on cygwin.  You'll need to either make a separate test for that environment (and skip this test when we are on cygwin), or add an alternative .good file for that setting.  Since we would like to support cygwin eventually I recommend making a separate test as a future, with the test cases you removed, their expected output in the .good file, and the error message as your .bad file for that test.  The future should mention that we want to support isAbsPath on Windows

Reply:
This future would also need to limit itself to only run on Windows

Reply:
@lydia-duncan  Can I make one separate file for cygwin testcases and their output so that when it will get implemented then it may get tested on it .

Reply:
Yup!

Reply:
There should be examples of tests that only use cygwin in the test directory, or that exclude cygwin.

Reply:
okay I will make one separate directory for cygwin testcase and testoutput file

Reply:
@lydia-duncan  I have added separate file for the test cases . Kindly suggest other necessary changes that should be done

Reply:
I'm not seeing a skipif to limit these tests to only certain configurations, as I asked (like [this file](https://github.com/chapel-lang/chapel/blob/master/test/studies/shootout/k-nucleotide/bradc/SKIPIF), except the file name should be testcases_isAbsPath.skipif).

I also asked for the cygwin test to have a .future and a .bad file (see [here](https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/TestSystem.rst#futures-a-mechanism-for-tracking-bugs-feature-requests-etc) for details on what that means).  The new test will cause noise in our nightly runs without these files.

Reply:
@lydia-duncan I have added **skipif , .future and .bad file** . Kindly review it and suggest any further changes that should be done .